keps/sig-cli/3515-kubectl-explain-openapiv3/kep.yaml

```diff
@@ -0,0 +1,26 @@
+id: "3515"
+name: kubectl-explain-openapiv3
+title: Kubectl Explain OpenAPIv3
+kep-number: 3515
+authors: ['@alexzielenski']
+owning-sig: sig-cli
+participating-sigs: [sig-api-machinery, sig-cli]
+reviewers:
+  - "@KnVerey"
+  - "@seans3"
+  - "@apelisse"
+approvers:
+  - "@KnVerey"
+  - "@seans3"
+creation-date: "2022-09-14"
+last-updated: v1.19
+status: implementable
+stage: alpha
+latest-milestone: "1.26"
+milestone:
+    alpha: "1.26"
+    beta: "1.28"
+    stable: "1.29"
+feature-gates: []
+disable-supported: false
```

Comment:
I think this should be true? Per [this commit](https://github.com/kubernetes/enhancements/commit/4e3548d31285175968ec180395fa8dbf44d82ec9): 

> Can the feature be disabled once it has been enabled (i.e. can we rollback
  the enablement)

> The name is intended to refer to rolling back the enablement of the
feature, but can cause confusion. It can be mixed up with what the
consequences of rolling back the version are, rather than just the
enablement.